script/parse-nav.ts

```diff
@@ -1,11 +1,15 @@
 import * as cheerio from 'cheerio'
-import * as path from 'path'
-import * as fs from 'fs'
-const i18n = require('../')
-const locales = Object.keys(i18n.locales)
+// UNSAFE IMPORT BLOCK
+// DONT CHANGE IMPORT TO `import { docs, locales } from '../dist'`
+import * as i18nDocs from '../dist/docs.json'
+import * as i18nLocales from '../dist/locales.json'
```

Comment:
Why is it unsafe?

Reply:
Now that file doesn't work very well with the new building process, in `build-module` we generate `index.js` file what's contains all exported imports including that file. On the moment of finishing the main building that file doesn't exist and Node reports that are not able to require `navs.json` file, what's generating only now.

That's unsafe because the final `index.js` is not usable at this moment, but should be used here. In da future, this parser should be moved to the main build module process were easy to work with them.